From: Thomas Preud'homme <thomas.preudhomme@celest.fr>
Date: Tue, 4 May 2010 13:15:01 +0200
Subject: [PATCH] Include config.mak if existing

Include config.mak only if existing to allow clean and distclean target to be
used even before configure has been called.

Origin: vendor
Forwarded: no
Last-Update: 2010-05-04
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index cf38acc..cf62242 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@
 #
 
 TOP ?= .
-include $(TOP)/config.mak
+-include $(TOP)/config.mak
 
 CFLAGS+=-Wall
 CFLAGS_P=$(CFLAGS) -pg -static -DCONFIG_TCC_STATIC
-- 
